### PR TITLE
chore: replace omit

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     内部重构，不用写 changelog      |
| 🇨🇳 Chinese |        内部重构，不用写 changelog   |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
替换后发现了一些冗余代码清理了，还有一些不知道干啥的先保留了。

File: components/auto-complete/index.tsx [hunk 1]
```diff
@@ -115,7 +116,10 @@ const AutoComplete: React.ForwardRefRenderFunction<RefSelectProps, AutoCompleteP
             prefixCls={prefixCls}
             className={classNames(`${prefixCls}-auto-complete`, className)}
             mode={Select.SECRET_COMBOBOX_MODE_DO_NOT_USE as any}
-            getInputElement={getInputElement}
+            {...{
+              // Internal api
+              getInputElement,
+            }}
```
Comment (afc163) [inline components/auto-complete/index.tsx:122]: 这是？

Comment (zombieJ) [reply components/auto-complete/index.tsx:122]: AutoComplete 自定义输入组件的方法，是 Select 的内部接口。

Comment (afc163) [reply components/auto-complete/index.tsx:122]: 这样解构一下的目的是？

Comment (zombieJ) [reply components/auto-complete/index.tsx:122]: TS 不校验解构属性，否则需要把 Select as ComponentType<InternalSelectProps> 再添加这个属性。比较蛋疼。

**Pull request merged**